cypress.config.stage.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-undef */
+import { defineConfig } from 'cypress';
+import baseConfig from './cypress.config';
+
+const envOverride = {
+  // TODO: remove this WORKAROUND for https://github.com/cypress-io/cypress/issues/20647,
```

Comment:
ainda necessário esse comentário? Se não for, acredito que podemos remover pois  essa estrutura com arquivos de config já identifica as configurações específicas por ambiente.